public/usage-examples/terminal/read_char/read_char-1-ASCII-checker-oop.cs

```diff
@@ -0,0 +1,30 @@
+﻿using SplashKitSDK;
+
+namespace Program
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            SplashKit.WriteLine("Check the ASCII value of a character.");
+            SplashKit.WriteLine("Press 'q' to quit, or type any character to see its ASCII value.");
+
+            while (true)
```

Comment:
Remove while(true) [per this comment](https://github.com/thoth-tech/splashkit.io-starlight/pull/269#issuecomment-2511186618)